### PR TITLE
Restart Workload feature

### LINKED ISSUE #735: implement restart functionality for deployments statefulsets and daemonsets
**Description**

Daemonsets, deployments and stateful sets can be restarted by issuing `kubectl rollout restart ... `. A similar functionality would be really useful.

**Reasons**

Developers need to restart workloads.

**Attachments**

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Closes #735

### CODE REVIEW

**Review by akucharska — COMMENTED** [review]
Nice. I added one question below

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -480,6 +481,8 @@ common:
     parse-error: Parse error, previous valid input will be used.
     resource-name-error: Requested resource with this name not found in the selected Namespace.
     resource-namespace-error: Requested Namespace not found.
+    restart-failure: Restarting failed.
+    restart-success: Workload restarted.
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply core/src/i18n/en.yaml:485]: Possibly... I'll ask a TW.

[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply core/src/i18n/en.yaml]: 2:1, I've changed the translations.

**Review by akucharska — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**